docs/Node Operators/configuration-variables.md

```diff
@@ -511,6 +511,13 @@ This setting tells the Chainlink node to log SQL statements made using the defau
 
 Determines the log file's max size in megabytes before file rotation. Having this not set will disable logging to disk. If your disk doesn't have enough disk space, the logging will pause and the application will log errors until space is available again.
 
+Any value should be suffixed with a unit like: "5120mb" (for 5,120 Megabytes). The unit suffixes need to be one of the following:
```

Comment:
We also support `b` for just bytes, but I think nobody will use it.
Also `Any value should be suffixed` is wrong, because if no suffix is provided we treat the value as bytes.

Reply:
The open nurse PR has a version of this too: https://github.com/smartcontractkit/documentation/pull/496

Reply:
@pinebit thanks, updated in 9c2d6c07d4e529fc62db848fc810e71f9a7702cb